frontend/src/views/sql-editor/TabListContainer.vue

```diff
@@ -278,7 +287,7 @@ onMounted(async () => {
 // add listener to confirm confrim if close the tab.
 onMounted(() => {
   window.onbeforeunload = () => {
-    return "false";
+    // return "false";
```

Comment:
Maybe we can just remove the `onMounted`

Reply:
Sry that's for debug only

Reply:
Updated, PTAL.